packages/config/config/devices/0x0258/nas-wr01z.json

```
@@ -0,0 +1,200 @@
+// Shenzhen Neo Electronics Co., Ltd. NAS-WR01Z
+// Wall Plug Switch
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NAS-WR01Z",
+	"description": "Wall Plug Switch",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x1087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x2087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x1483"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Meter Report Enable",
```

```suggestion
			"label": "Send Meter Reports",
```
We're trying to use labels for which the options Enable/Disable make sense whereever possible.